server/test/accountCreation.test.js

```diff
@@ -0,0 +1,34 @@
+const accountCreationHandler = require('../models/handlers/accountCreation');
+
+ describe('Test to check valid account creation', () => {
+     beforeEach(() => {
+         jest.resetModules();
+     });
+//Create a test using Arrange, Act, Assert format
+     test('Checking if the account creation details are correct', async() => {
+         // Arrange
+         
+         /* In order to test, the tester should input these credentials*/
+         const email = 'victorhugo@gmail.com';
+         const password = 'victorlovespoetry';
+         const testUser = {
+            type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
```

Comment:
This comment is really great! I would recommend moving it to the accountCreation handler so others will easily see it and it does not get lost when more test code is created.

Reply:
Hi Amy, 
Thank you for suggesting that! 
**My action:** I've added this comment to the accountCreation handler. I agree, this would avoid any confusion between frontend and backend the naming convention for testers. 
You can now see it in line 8 of server/routes/publicAccountCreation.js

Reply:
Hi Nancy, that's perfect. Thank you for doing that!